base/applications/regedit/regedit.c

```diff
@@ -119,6 +119,22 @@ void WINAPIV error_exit(unsigned int id, ...)
     exit(0); /* regedit.exe always terminates with error code zero */
 }
 
+#ifdef __REACTOS__
+void WINAPIV error_dont_exit(unsigned int id, ...)
+{
+    WCHAR fmt[256], text[512], title[64];
+    va_list va;
+
+    LoadStringW(hInst, IDS_APP_TITLE, title, _countof(title));
```

Comment:
I would propose to strip variable title, the LoadString() call for that and pass NULL instead into the MessageBoxW().

Reply:
OK.